client/src/app/s-degree-plan-editor/s-degree-plan-editor.component.html

```diff
@@ -27,7 +27,9 @@
               />
               <div class="semester-card">
                 <div class="semester-card-term-and-status">
-                  <span> {{ semester.term }} {{ semester.year }}</span>
+                  <span>
```

Comment:
check if semester term and year are defined with *ngIf in this span

Reply:
likewise, add to line 13 
<span class="year-range">{{ year.beginning }} - {{ year.ending }}</span>

and line 180 in the existing ngIf
<button
              *ngIf="year.newSemesterWidget.active === false"
              mat-button
              (click)="toggleAddingSemester(yearIndex)"
              class="add-semester-font-clr w100"
            >
              + ADD A SEMESTER FOR {{ year.beginning }} - {{ year.ending }}
            </button>

Reply:
Added